recipes/nr/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "nr" %}
+{% set version = "1.3.5" %}
+{% set sha256 = "43f4b1df4168647604242209d4551085977262d0f4b9030430363b2cd75c4374" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/NiklasRosenstein/py-nr/archive/v1.3.5.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - pandoc
+  run:
+    - python
+
+test:
+  commands:
+    - echo 'skip tests (conda-build bugged)'
```

Comment:
This is not a good enough test.  There should be an existence test at the very least. see https://github.com/conda-forge/conda-forge.github.io/pull/283/files